assets/sass/components/key-metrics/_googlesitekit-key-metrics-widgets.scss

```diff
@@ -28,6 +28,16 @@
 				@include mdc-layout-grid-cell-span_("phone", 4, $gutter);
 			}
 		}
+
+		.googlesitekit-widget-area-header__title {
+
+			align-items: center;
+			display: flex;
+
+			.googlesitekit-new-badge {
+				margin-left: 8px;
```

Comment:
The gap between the `Key metrics` title and the `New` badge seems lesser than the Figma design. Can we make it `10px` or `12px`?

**Figma design**

<img width="404" alt="Screenshot 2023-08-24 at 10 03 40 PM" src="https://github.com/google/site-kit-wp/assets/24408261/e6c50889-afcb-40ca-af59-3b799b90e8a3">

**Current implementation**

<img width="396" alt="Screenshot 2023-08-24 at 10 03 21 PM" src="https://github.com/google/site-kit-wp/assets/24408261/405bc2ef-e66f-4e8e-a393-f831e49df31a">

Reply:
Good call, I've made the change 👍🏻